package.json

```diff
@@ -30,6 +31,8 @@
     "0.0.0"
   ],
   "devDependencies": {
+    "@ember/jquery": "^0.5.2",
```

Comment:
Adding this is 👍, but it will mean that you need to tweak all of your Ember < 2.8 ember-try scenarios to bring jQuery@2.

Reply:
Checkout [this 2.4 scenario](https://github.com/ember-cli/ember-cli-htmlbars-inline-precompile/blob/92dbe970e39b88a039ced73d379dbc2a3dbd8e27/config/ember-try.js#L32-L52):

```js
        {
          name: 'ember-lts-2.4',
          env: {
            EMBER_OPTIONAL_FEATURES: JSON.stringify({ 'jquery-integration': true }),
          },
          bower: {
            dependencies: {
              ember: 'components/ember#lts-2-4',
            },
            resolutions: {
              ember: 'lts-2-4',
            },
          },
          npm: {
            devDependencies: {
              '@ember/jquery': '^0.5.1',
              'ember-source': null,
              'jquery': '^2.2.4',
            },
          },
        },
```

Reply:
Thanks! Just made the changes. Hopefully tests will pass now 🤞